### ISSUE TITLE
Digital Catalog (Top Charts)

### ISSUE THREAD

**Issue by cbj4074** [indiehd/web-api#17]
Get Top N Artists (n Best-Selling)
  - sales grouped by Artist

Get Top N Labels (n Best-Selling)
  - sales grouped by Label

Get Top N Albums (n Best-Selling)
  - sales grouped by Album

**Comment by poppabear8883** [comment]
Is the following the same thing as referenced here ?

![image](https://user-images.githubusercontent.com/7263458/58723015-f9301b00-83a6-11e9-948c-74177e067a52.png)

**Comment by cbj4074** [comment]
Yes, that is the first item, `Get Top N Artists (n Best-Selling)`, where `n = 5`.

**Comment by poppabear8883** [comment]
Ref: https://github.com/indiehd/web-ui/pull/10

- [ ] Tests 
- [ ] API Endpoints
 * `/top/artists`
 * `/top/songs`
 * `/top/albums`

- [ ] TopController, TopRepository
- [ ] Top Api Resources

**Comment by cbj4074** [comment]
#138 is related to this.

It may not necessarily need to be completed before we can implement this, because we can probably just seed the sales data, but we'll likely need to build-out at least part of what's required to satisfy that other Issue before we can fully-implement this one.
